### PR TITLE
Fetch last 3 blog posts from Ghost and display them on the home page.

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

**Comment by mmilenkovic** [comment]
@AlbertoElias @elias-garcia Cleaned up. All should be well.

**Comment by mmilenkovic** [comment]
contentApiKey regenerated. Secret updated. Security restored. 🥳

File: src/components/home/home.component.jsx [hunk 1]
```diff
@@ -206,15 +207,15 @@ const Home = () => {
             <div className={classes.col2}>
               <h3 className={classes.blogTitle}>
                 <a
-                  href='https://blog.hermez.io/hermez-bug-bounty-program/'
+                  href={blogPosts[0].url}
```
Comment (AlbertoElias) [inline src/components/home/home.component.jsx]: Couldn't all this happen in a for loop so you only repeat the blog elements once?

Comment (elias-garcia) [reply src/components/home/home.component.jsx]: yeah right! updated

File: src/components/home/home.component.jsx [hunk 2]
```diff
@@ -200,63 +200,35 @@ const Home = ({ blogPosts }) => {
...
-          <div className={classes.row}>
-            <span className={classes.divider} />
-          </div>
+          {
+            blogPosts.map((blogPost, index) => (
+              <>
+                <div className={classes.row} key={index}>
+                  {index === 0 && (
+                    <div className={`${classes.col3} ${classes.blogHeaderCol}`}>
+                      <Title>Blog</Title>
```
Comment (AlbertoElias) [inline src/components/home/home.component.jsx:209]: why not just take this outside of the map?

Comment (mmilenkovic) [reply src/components/home/home.component.jsx:209]: It needs to be in the col3.

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**